packages/yoastseo/src/scoring/assessments/readability/SubheadingDistributionTooLongAssessment.js

```diff
@@ -148,11 +159,11 @@ class SubheadingsDistributionTooLong extends Assessment {
 		const currentConfig = this._config;
 		const languageSpecificConfig = researcher.getConfig( "subheadingsTooLong" );
 		// Check if a language has a default cornerstone configuration.
-		if ( currentConfig.cornerstoneContent === true && languageSpecificConfig.hasOwnProperty( "cornerstoneParameters" ) ) {
+		if ( currentConfig.cornerstoneContent === true && Object.hasOwn( languageSpecificConfig,  "cornerstoneParameters" ) ) {
```

Comment:
Per [this documentation](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/hasOwnProperty), it is recommended to use `Object.hasOwn`

Reply:
Good that you also decreased the warnings in a58880bad2aab5f9473a1685c03f6bf81e3b7beb!